rhel/repositoryscanner_test.go

```diff
@@ -55,6 +60,23 @@ func TestRepositoryScanner(t *testing.T) {
 	srv := httptest.NewServer(mux)
 	defer srv.Close()
 
+	esrv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		t.Fatal("external http request invoked when none was expected")
+	}))
+
+	f, err := os.CreateTemp("", "repository-2-cpe.json")
+	if err != nil {
+		t.Fatal("trying to create repository-2-cpe.json for FromMappingFile test", err)
+	}
+	defer os.Remove(f.Name())
+
+	if _, err := f.Write(mappingDataBytes.Bytes()); err != nil {
+		t.Fatal(fmt.Sprintf("trying to write %s for FromMappingFile test", f.Name()), err)
```

Comment:
`t.Fatalf` is useful here (and in the next block)

Reply:
🤦 I swear I've written Go before